### PR TITLE
Fix release process and remove matches

### PR DESCRIPTION
**Pull request by anthonyroach**
Only master branch pushes should do a release.

Also removing matches since it is still being solidified in
another PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,3 +46,8 @@ abac.policyRequiresAttribute(policy, attribute);
 ```
 
 See unit tests in `/test` folder - many good examples.
+
+## Release process
+
+Increment the version in `package.json`, make a PR, merge the PR,
+and then finally tag mater with a tag like `v4.5.3`.
```
Comment (bishopb) [inline README.md]: Nit: `mater` should be `master`.

**Review by bishopb — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (anthonyroach) [reply README.md]: fixed with --amend

File: test/enforce.test.js [hunk 2]
```diff
@@ -783,28 +783,6 @@ test('rules can use notIn with referenced values', t => {
...
-
-  t.true(enforce('readData', policy, {object: {value: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa!'}}), 'enforce string');
-  t.false(enforce('readData', policy, {object: {value: ''}}), 'enforce empty string');
-  t.false(enforce('readData', policy, {object: {value: undefined}}), 'enforce undefined');
-  t.false(enforce('readData', policy, {object: {value: null}}), 'enforce null');
-  t.false(enforce('readData', policy, {object: {value: 1}}), 'enforce number');
-  t.false(enforce('readData', policy, {object: {value: ' '}}), 'enforce space');
-});
-
 test('rules can use matches operator with no value', t => {
```
Comment (sgrogg72) [inline test/enforce.test.js:786]: remove this test too

Comment (anthonyroach) [reply test/enforce.test.js]: done

**Review by jjbeavers-lifeomic — APPROVED** [review]

**Pull request merged**